service/test/unit/resources/test_backup_account_resource.py

```diff
@@ -41,3 +42,45 @@ def assert_200_when_user_logged_in(_):
 
         d.addCallback(assert_200_when_user_logged_in)
         return d
+
+    @patch('pixelated.resources.backup_account_resource.AccountRecovery')
+    def test_post_updates_recovery_code(self, mock_account_recovery_init):
+        mock_account_recovery = MagicMock()
+        mock_account_recovery_init.return_value = mock_account_recovery
+        mock_account_recovery.update_recovery_code.return_value = defer.succeed("Success")
```

Comment:
Since we are using "Success" as a result of the deferred, we should assert on the value as well? If the value is not being used in the actual code, then no need to pass a value here?